bci_main.py

```diff
@@ -126,6 +124,12 @@ def _clean_up_session(display, daq, server):
     if server:
         server.stop()
 
+    # Close the display window
+    # NOTE: There is currently a bug in psychopy when attempting to shutdown
+    # windows when using a USB-C monitor. Putting the display close last in
+    # the sequence allows acquisition to properly shutdown.
+    display.close()
```

Comment:
I agree with moving. To prevent hanging screens, we might consider wrapping this and forcing a kill in another way if it fails to close. What error is thrown when it tries to close with the USB-C monitor? We can merge this as is but worth looking into.

Reply:
I don't think it ever actually throws an exception, it just hangs. The current workaround is to force with Ctrl-C. I think it only happens in fullscreen mode. Next time I'm in the lab I'll confirm this behavior. If I can duplicate with a small script I will report the issue to the psychopy project.